reana_commons/openapi_specifications/reana_server.json

```diff
@@ -1032,6 +1032,209 @@
         "summary": "Ping the server (healthcheck)"
       }
     },
+    "/api/quota": {
```

Comment:
@CameronMcClymont BTW please use a more descriptive commit log message, since "update specs" can mean anything... e.g. "feat(api): add user quota management endpoints" or some such.  (And you can use the same (or similar) message for r-server and r-commons so that people can make mental link where does this come from.)